docs/source/howto/installation.rst

```diff
@@ -223,24 +223,344 @@ the configuration directory ``~/project_a/.aiida`` will be used.
+If you already configured the computer and just want to adapt the computer configuration, you can run again
+
+.. code:: bash
+
+    verdi computer configure ssh COMPUTERNAME
+
+After the configuration, you should verify that AiiDA can connect to the computer successfully using:
+
+.. code:: bash
+
+    verdi computer test COMPUTERNAME
+
+
+.. _how-to:installation:running-on-supercomputers:avoiding-overloads:
+
+Avoiding overloads
+------------------
+
+If you submit to a supercomputer shared by many users (e.g., in a supercomputer center), be careful not to overload the supercomputer with too many jobs:
+
+  * keep the number of jobs in the queue under control (the exact number depends on the supercomputer: discuss this with your supercomputer administrators, and you can redirect them to :ref:`this page<how-to:installation:running-on-supercomputers:for_cluster_admins>` that may contain useful information for them).
+    While in the future `this might be dealt by AiiDA automatically <https://github.com/aiidateam/aiida-core/issues/88>`_,
+    you are responsible for this at the moment.
+    This can be achieved for instance by submitting only a maximum number of workflows to AiiDA, and submitting new ones only when the previous ones complete.
+
+  * Tune the parameters that AiiDA uses to avoid overloading the supercomputer with connections or batch requests.
+    For SSH transports, the default is 30 seconds, which means that when each worker opens a SSH connection to a computer, it will reuse it as long as there are tasks to execute and then close it.
+    Opening a new connection will not happen before 30 seconds has passed from the opening of the previous one.
+
+    We stress that this is *per daemon worker*, so that if you have 10 workers, your supercomputer will on average see 10 connections every 30 seconds.
+    Therefore, if you are using many workers and you mostly have long-running jobs, you can set a longer time (e.g., 120 seconds) by reconfiguring the computer with ``verdi computer configure ssh <COMPUTER_NAME>`` and changing the value
+    of the *Connection cooldown time* or, alternatively, by running::
+
+      verdi computer configure ssh --non-interactive --safe-interval <SECONDS> <COMPUTER_NAME>
+
+  * In addition to the connection cooldown time described above, AiiDA also avoids running too often the command to get the list of jobs in the queue (``squeue``, ``qstat``, ...), as this can also impact the performance of the scheduler.
+    For a given computer, you can increase how many seconds must pass between requests.
+    First load the computer in a shell with ``computer = load_computer(<COMPUTER_NAME>)``.
+    You can check the current value in seconds (by default, 10) with ``computer.get_minimum_job_poll_interval()``.
+    You can then set it to a higher value using::
+
+      computer.set_minimum_job_poll_interval(<NEW_VALUE_SECONDS>)
```

Comment:
We have decided in a separate discussion to always use explicit mark-up for code segments, see [here](https://github.com/aiidateam/aiida-core/wiki/Writing-documentation#code-segments).

Reply:
Ok, sorry, I had missed that - I'll fix this